src/components/app/nav.jsx

```diff
@@ -173,6 +173,7 @@ class AppNav extends Component {
           <span className='repo-owner'>{repoOwner}</span>
           {'/'}
           {repoNameItems}
+          <img className='avatar-image' src={'https://github.com/' + repoOwner + '.png'} alt='repo-owner'/>
```

Comment:
Is this a trick or supported by GitHub officially?

Reply:
I believe this is sort of a 'trick'. 
Got this from [StackOverflow](https://stackoverflow.com/questions/22932422/get-github-avatar-from-email-or-name)
This 'trick' is from this [video](https://youtu.be/jpMmboYadnA) posted by GitHub itself.
I haven't found any official documentation as I believe this is a 'trick' provided by GitHub itself.
I felt this would be a better way since we do not have to make additional API requests for GitHub profile logos.

Reply:
Cool trick.